reference/compiling_for_android.rst

```diff
@@ -16,93 +16,81 @@ Also, you still need to do all the steps mentioned in the
 :ref:`doc_exporting_for_android` tutorial before attempting your custom
 export template.
 
-Requirements
-------------
+A. Requirements
```

Comment:
I'm not for adding those, the table of contents does this job already. And it's inconsistent with the rest of the documentation, and impossible to maintain.